int32[] custom_class_ids         # Inception 모델로 분류된 클래스 ID 배열
string[] custom_class_names      # Inception 모델로 분류된 클래스 이름 배열
string target_user_name          # 선택된 사용자 이름